pkg/util/validation/errors.go

```diff
@@ -100,8 +100,15 @@ func newTooManyLabelsError(series []mimirpb.LabelAdapter, limit int) ValidationE
 }
 
 func (e tooManyLabelsError) Error() string {
+	metric := mimirpb.FromLabelAdaptersToMetric(e.series).String()
+	ellipsis := ""
+
+	if len(metric) > 200 {
+		ellipsis = "..."
```

Comment:
super-nit: we could use https://www.compart.com/en/unicode/U+2026. In utf-8 it's also 3 bytes, but it is shorter (one char on terminal) when printed :)

(note that duration-formatting also uses unicode symbol `µ` so there's precedent already)

Reply:
Good idea, changed in ceade5c